task_manager/users/templates/create_user.html

```diff
@@ -0,0 +1,19 @@
+{% extends "base.html" %}
+{% block content %}
+  {% load django_bootstrap5 %}
+  {% load i18n %}
+
+  <div class="container wrapper flex-grow-1">
+    <h1 class="my-4">{% trans "Signing up" %}</h1>
+
+    <form method="post">
+      {% csrf_token %}
+      {% bootstrap_form form %}
+      <input
```

Comment:
this form has fewer requirements in the hexlet example, see:

hexlet - 
<img width="540" alt="image" src="https://github.com/AlexMomot-717/python-project-lvl4/assets/45393914/bb596e57-1973-49fa-be51-e52bd93f3b35">

your app - 
<img width="558" alt="image" src="https://github.com/AlexMomot-717/python-project-lvl4/assets/45393914/eea9a6de-37b9-460b-9c25-1568267f071a">